index.html

```diff
@@ -0,0 +1,123 @@
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e8963cdebe.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Nairobi Classical Festival</title>
+</head>
+<body>
+    <main>
+        <header>
+            <div class="desktop-only top-header">
+                <i class="fa fa-facebook" aria-hidden="true"></i>
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+                <p>Translate</p>
+                <p>Logout</p>
+            </div>
+
+            <div class="desktop-only bottom-header">
+                <div class="project-logo">
+                    <img src="images/main-logo.svg" alt="">
+                </div>
+                <div class="desktop-header-content">
+                    <a href="about.html" class="about-header">About</a>
+                    <p>News Letter</p>
+                    <p>Concert Program</p>
+                    <p class="attend">Attend</p>
+                </div>
+            </div>
+
+            <div class="top-menu">
+                <img src="images/menu.png" alt="">
+            </div>
+            <div class="header-content">
+                <p class="hello">"Hello! Musical World!"</p>
+                <h1>Nairobi Classical Concert 2022</h1>
+                <p class="intro-text">A once in a lifetime classical concert coming to Nairobi, Kenya in November, Bringing together
+                    musical prodigies and maestros from all over the world! </p>
+                <p class="date">2022.11.08 (TEU) ~ 11 (FRI)</p>
+                <small>@ Kenyatta University Auditorium </small>
+            </div>
+        </header>
+
+          <!-- Mobile menu  -->
+
+        <div class="mobile-menu">
+            <p class="cancel">&times;</p>
+            <a href="#"><p>Home</p></a>
+            <a href="about.html"><p>About</p></a>
+        </div>
+
+        <!-- Program Section -->
+        <section class="main-program">
+            <h2>Concert Program</h2>
+            <div class="program-cards">
+                <div class="card">
+                    <img src="images/event.png" alt="">
+                    <h3>Concert</h3>
+                    <p>Mozart Violin Cocerto in G major Presented by American Violinist Ray Chen</p>
+                </div>
+
+                <div class="card">
+                    <img src="images/event.png" alt="">
+                    <h3>Concert</h3>
+                    <p>Mozart Violin Cocerto in G major Presented by American Violinist Ray Chen</p>
+                </div>
+
+                <div class="card">
+                    <img src="images/event.png" alt="">
+                    <h3>Concert</h3>
+                    <p>Mozart Violin Cocerto in G major Presented by American Violinist Ray Chen</p>
+                </div>
+
```

Comment:
- [ ] Kindly add more articles, the total should be 5 as suggested in the video.
 - [ ] Kindly follow the spacing as suggested in the video.

Reply:
done